addons/l10n_my_ubl_pint/views/res_partner_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="view_partner_form_my_pint" model="ir.ui.view">
+        <field name="name">res.partner.form.inherit.l10n_my_pint</field>
+        <field name="inherit_id" ref="account.view_partner_property_form"/>
+        <field name="model">res.partner</field>
+        <field name="priority">100</field>
+        <field type="xml" name="arch">
+            <xpath expr="//field[@name='vat']" position="after">
+                <field name="sst_registration_number" invisible="country_code != 'MY'"/>
```

Comment:
I never saw this image 0.0
I decided to put it on the company since it's where it's taken from. I don't mind adding it to the partner view too if you prefer.

Reply:
It's from the task: https://www.odoo.com/web#id=3440503&cids=1&model=project.task&view_type=form :D

Reply:
If we're not sure, I'd say let's keep it like that, we can always add it to the view if necessary later :)

Reply:
I already put it on the partner view too ^^
IMO both are fine, it doesn't hurt to have it on both side. Makes it easier for users who are used to setting these things on the partner view directly for example.